portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,64 @@
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+</head>
+
+<body>
+    <div class="top-nav-bar">
+        <ul>
+            <li><a href="index.html"> HOME </a> </li>
+            <li> <a href="about.html">ABOUT </a></li>
+            <li> PROJECTS </li>
+            <li> BLOG </li>
+            <li> CONTACT </li>
+        </ul>
+    </div>
+
+<div class="parent-content">
+        <div class="wrapper">
+            <h1>
+                ABOUT ME
+            </h1>
+            <img src="/images/IMG_0111.jpeg"><br>
+            <div class="content-left">
+                <p>
+                    My name is Kidist Alemu and I am a sophomore at Harvard studying Computer Science.
+                    I was born and raised in Addis Ababa, Ethiopia but I moved to the United States for College

```

Comment:
Add a period at the end of the sentence.

Reply:
Updated!